<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UtilTest</class>
 <widget class="QWidget" name="UtilTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>426</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Input file:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="ctkPathLineEdit" name="inputFileSelector">
     <property name="settingKey">
      <string>RawImageGuessInputFile</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="imageSkipLabel">
     <property name="text">
      <string>Header size:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="ctkSliderWidget" name="imageSkipSliderWidget">
     <property name="toolTip">
      <string>Set threshold value for computing the output image. Voxels that have intensities lower than this value will set to zero.</string>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="singleStep">
      <double>1.000000000000000</double>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="maximum">
      <double>2500.000000000000000</double>
     </property>
     <property name="value">
      <double>0.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="imageSizeXLabel">
     <property name="text">
      <string>X dimension:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="ctkSliderWidget" name="imageSizeXSliderWidget">
     <property name="toolTip">
      <string>Set threshold value for computing the output image. Voxels that have intensities lower than this value will set to zero.</string>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="singleStep">
      <double>1.000000000000000</double>
     </property>
     <property name="minimum">
      <double>1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>5000.000000000000000</double>
     </property>
     <property name="value">
      <double>200.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="imageSizeYLabel">
     <property name="text">
      <string>Y dimension:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="ctkSliderWidget" name="imageSizeYSliderWidget">
     <property name="toolTip">
      <string>Set threshold value for computing the output image. Voxels that have intensities lower than this value will set to zero.</string>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="singleStep">
      <double>1.000000000000000</double>
     </property>
     <property name="minimum">
      <double>1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>2500.000000000000000</double>
     </property>
     <property name="value">
      <double>200.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="imageSizeZLabel">
     <property name="text">
      <string>Z dimension:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="ctkSliderWidget" name="imageSizeZSliderWidget">
     <property name="toolTip">
      <string>Set threshold value for computing the output image. Voxels that have intensities lower than this value will set to zero.</string>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="singleStep">
      <double>1.000000000000000</double>
     </property>
     <property name="minimum">
      <double>1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>2500.000000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Pixel type:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="ctkComboBox" name="pixelTypeComboBox">
     <item>
      <property name="text">
       <string>8 bit unsigned</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8 bit signed</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>16 bit unsigned</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>16 bit signed</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>float</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>double</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Header:</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="2">
    <widget class="QTextEdit" name="textEdit"/>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="ctkCheckablePushButton" name="updateButton">
     <property name="text">
      <string>Update</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Output Volume:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="qMRMLNodeComboBox" name="outputVolumeNodeSelector">
     <property name="toolTip">
      <string>Pick the output to the algorithm.</string>
     </property>
     <property name="nodeTypes">
      <stringlist>
       <string>vtkMRMLScalarVolumeNode</string>
      </stringlist>
     </property>
     <property name="showChildNodeTypes">
      <bool>false</bool>
     </property>
     <property name="noneEnabled">
      <bool>true</bool>
     </property>
     <property name="addEnabled">
      <bool>true</bool>
     </property>
     <property name="removeEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCheckablePushButton</class>
   <extends>ctkPushButton</extends>
   <header>ctkCheckablePushButton.h</header>
  </customwidget>
  <customwidget>
   <class>ctkComboBox</class>
   <extends>QComboBox</extends>
   <header>ctkComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>ctkPathLineEdit</class>
   <extends>QWidget</extends>
   <header>ctkPathLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>ctkPushButton</class>
   <extends>QPushButton</extends>
   <header>ctkPushButton.h</header>
  </customwidget>
  <customwidget>
   <class>ctkSliderWidget</class>
   <extends>QWidget</extends>
   <header>ctkSliderWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
